docs/tr/themes.md

```diff
@@ -0,0 +1,29 @@
+# Tema
+
+Flarum'u olabildiğince güzel hale getirmek için çok çalışmış olsak da, her topluluk muhtemelen arzu ettikleri stile uyacak bazı ince ayarlar/değişiklikler yapmak isteyecektir.
+
+## Yönetici Gösterge Paneli
+
+[Yönetici gösterge tablosu](../admin.md)'nda Görünüm sayfası, forumunuzu özelleştirmeye başlamak için harika bir ilk yerdir. Burada yapabilirsin:
+
+- Tema renklerini seçin
+- Karanlık modu ve renkli üstbilgi değiştir
+- Bir logo ve favicon yükleyin (tarayıcı sekmelerinde gösterilen simge)
+- Özel üstbilgiler ve altbilgiler için HTML ekleyin
+- Öğelerin görüntülenme şeklini değiştirmek için [özel LESS/CSS](#css-theming) ekleyin
+
+## CSS Tema Oluşturma
+
+CSS, tarayıcılara bir web sayfasının öğelerinin nasıl görüntüleneceğini söyleyen bir stil sayfası dilidir. Renklerden yazı tiplerine, eleman boyutuna ve konumlandırmadan animasyonlara kadar her şeyi değiştirmemizi sağlar. Özel CSS eklemek, Flarum kurulumunuzu bir temayla eşleşecek şekilde değiştirmenin harika bir yolu olabilir.
+
+Bir CSS eğitimi bu dokümantasyonun kapsamı dışındadır, ancak CSS'nin temellerini öğrenmek için birçok harika çevrimiçi kaynak vardır.
```

Comment:
"Doküman" yerine "belge" diyebiliriz.

Reply:
Thanks, fixed it.

- https://github.com/flarum/docs/pull/154/commits/bb0c788c46394d8acac0a45c1172ffc627dfd33e